File: examples/before/sample.js
// Example JavaScript code with deprecated API
const oldApi = require('old-api');

function main() {
    const client = oldApi.createClient({ key: 'secret' });
    const user = oldApi.getUser(123);
    const posts = oldApi.fetchPosts(123);
    return { user, posts };
}

main();

File: examples/before/sample.py
# Example Python code with deprecated API
from old_api import Client

def main():
    client = Client(api_key="secret")
    user = old_api.get_user(user_id=123)
    posts = old_api.fetch_posts(user_id=123)
    return user, posts

if __name__ == "__main__":
    main()
